test/functional/terminal/scrollback_spec.lua

```diff
@@ -416,7 +438,11 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+    if iswin() then
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\r')
```

Comment:
@janlazo Thanks. Finally I understood the meaning you said. It was talking about whether the escape sequence of powershell is <code>&#x60;</code>, so <code>&#x60;n</code> can not be used. Since it is a special character of vim that has meaning here, it is no use to use the escape sequence of powershell. I am not confident as to whether I can explain it well because my poor English. However, I think that you can understand from the next execution result.

```
:echo map(split("\r`r", '\Z'), {idx, val -> char2nr(val)})
```